### PR TITLE
Add test case to verify what can match organizations/unknown

### PR DESCRIPTION
**Pull request by iyabchen**

### CODE REVIEW

File: pkg/gcptarget/gcptarget_test.go [hunk 1]
```diff
@@ -118,6 +118,38 @@ var matchTests = []reviewTestData{
...
+	{
+		name: "organizations/** can match organizations/unknown",
+		match: map[string]interface{}{
+			"ancestries": []interface{}{"organizations/**"},
+		},
+		ancestryPath: "organizations/unknown/folders/1221214",
+		wantMatch:    true,
+	},
+	{
+		name: "organizations/* can NOT match organizations/unknown",
```
Comment (melinath) [inline pkg/gcptarget/gcptarget_test.go:130]: Could you add a test for whether it matches `organizations/unknown` if the ancestryPath is `"organizations/unknown"` without any additional parts? (I think it _should_ but I could be wrong.)

**Review by melinath — CHANGES_REQUESTED** [review]
One small change request. Also, it would be great to be able to provide `organizations/unknown` as the match parameter. I know you mentioned that gave you an error - could you share the error message or traceback if there is one?

**Comment by iyabchen** [comment]
> One small change request. Also, it would be great to be able to provide `organizations/unknown` as the match parameter. I know you mentioned that gave you an error - could you share the error message or traceback if there is one?

Since unknown is not *, or **, or numbers, it throws error from this function
https://github.com/GoogleCloudPlatform/config-validator/blob/master/pkg/gcptarget/gcptarget.go#L262

File: pkg/gcptarget/gcptarget_test.go (hunk 1 above)
Comment (iyabchen) [reply pkg/gcptarget/gcptarget_test.go]: Added. Yes, it can match organizations/x, but not organizations/x/folders/y

**Pull request merged**